### PR TITLE
Going fshing

### PR DESCRIPTION
**Pull request by jvitrifork**
Full transition to FSH

### CODE REVIEW

File: input/fsh/instances.fsh [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+Usage: #example
+* name[0].use = #temp
+* name[0].family = "Ukendt"
+* name[0].given[0] = "Anders"
+* identifier[0].system = "http://rn.dk"
+* identifier[0].value = "0706830VJ9"
+* gender = #male
+* birthDate = "1983-06-07"
+
+Instance: administrative-gender-supplement
```
Comment (tmh-mjolner) [inline input/fsh/instances.fsh:85]: Why is this and other code systems / value sets defined as instances and not the native CodeSystem / ValuSet in FSH?

File: input/fsh/profiles.fsh [hunk 2]
```diff
@@ -0,0 +1,108 @@
+Profile: AuthorizationIdentifier
```
Comment (tmh-mjolner) [inline input/fsh/profiles.fsh:1]: I would in general prefer to have each profile in separate files (comment also apply to extensions, invariants, ...)

Comment (jvitrifork) [reply input/fsh/profiles.fsh:1]: ok - so ```file-per-definition``` https://fshschool.org/docs/gofsh/running/#style  ?

File: sushi-config.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# ╭──────────────────────────────────────ImplementationGuide───────────────────────────────────────╮
+# │  The properties below are used to create the ImplementationGuide resource. For a list of       │
+# │  supported properties, see: https://fshschool.org/sushi/configuration/                         │
+# ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
+id: dk.fhir.ig.dk-core
+canonical: http://hl7.dk/fhir/core
+name: DK_Core
+status: draft
+version: 0.1.0
```
Comment (tmh-mjolner) [inline sushi-config.yaml:9]: Version 0.9.0 -> 0.1.0 seems wrong (fix before merge)

File: input/fsh/profiles.fsh (hunk 2 above)
Comment (tmh-mjolner) [reply input/fsh/profiles.fsh:1]: Dont't know GoFSH that well

Comment (jvitrifork) [reply input/fsh/profiles.fsh]: Its now split across multiple files ... I can't say that I favour that layout ...

File: input/fsh/instances.fsh (hunk 1 above)
Comment (jvitrifork) [reply input/fsh/instances.fsh]: because its from the supplement

**Pull request merged**